Gemfile.lock

```diff
@@ -83,4 +105,4 @@ DEPENDENCIES
   waterdrop!
 
 BUNDLED WITH
-   2.5.16
+   2.6.2
```

Comment:
If we upgrade Bundler, let's go with the latest version, `2.6.5`.

Reply:
nah, this should be done by renovate, with me it's just alongside